src/boot/efi/boot.c

```diff
@@ -1204,6 +1219,19 @@ static void config_defaults_load_from_file(Config *config, CHAR8 *content) {
                         err = parse_boolean(value, &config->reboot_for_bitlocker);
                         if (EFI_ERROR(err))
                                 log_error_stall(L"Error parsing 'reboot-for-bitlocker' config option: %a", value);
+                }
+
+                if (strcmpa((CHAR8 *)"auto-enroll-keys", key) == 0) {
+                        err = parse_boolean(value, &config->auto_enroll_keys);
+                        if (EFI_ERROR(err))
+                                log_error_stall(L"Error parsing 'auto-enroll-keys' config option: %a", value);
+                        continue;
+                }
+
+                if (strcmpa((CHAR8 *)"i-might-brick-my-machine-please-auto-enroll-keys", key) == 0) {
```

Comment:
quite frankly, going overboard with the naming here I am not a fan of. I think if this is called "force" it already sounds dangerous enough.

Reply:
I think this might try to be in line with the `--yes-this-might-brick-my-machine` of @Foxboron's sbctl. 

I think it is sensible to name this with a clear warning of bricking. "force" might be too weak for this. (I'm imaging the usual anti-systemd crowd shouting from the sidelines)

Reply:
Well, if it was up to me, I'd just go with force, however like other people mentioned, there is a very real risk of soft-bricking and I thought it would be appreciated to be extremely clear to the user that this is a risky operation, arguably riskier that just wiping their disk or other "standard" risky action.

I indeed took example on the similarly named option from sbctl by @Foxboron. I hope @Foxboron will not mind me borrowing it, I felt the option was aptly named.

Reply:
dunno, if this is opt-in, is called "force" and not just "yes", and comes with documentation that shouldbe fine.

I mean, there are myriads of ways you can brick your system, I am not a fan of giving this specific way such a theatrical name.

Yes: opt-in is good. Yes: not just called this "yes" or "on", but something tougher such as "force" is good. And yes: documenting the effects is also good. But theatrics, I am not convinced of.